emsdk_manifest.json

```diff
@@ -1597,6 +1625,12 @@
     "version_filter": [
       ["%precompiled_tag64%", ">", "1.37.22"]
     ]
+  },
+  {
+    "version": "upstream-%waterfall-lkgr%",
+    "bitness": 64,
+    "uses": ["clang-upstream-%waterfall-lkgr%-64bit", "emscripten-upstream-%waterfall-lkgr%-64bit", "binaryen-upstream-%waterfall-lkgr%-64bit"],
+    "os": "linux"

```

Comment:
There is an implicit feature that "latest listed SDK is the newest stable version" that comes from https://github.com/juj/emsdk/blob/master/emsdk#L1463, so this added sdk section should probably be located after this line: https://github.com/jgravelle-google/emsdk/blob/f221912051bf1e7e92b38ac5655957c5f03af7f3/emsdk_manifest.json#L1219 so that `emsdk install latest` will not start downloading these waterfall builds instead of stable precompiled builds.

Reply:
Hmm, we do `install latest` in the test here, and it still downloaded the proper one, this didn't seem to mess it up?

Reply:
Yeah, the test here runs on linux on travis - which seems like what should have broken?